2024-08-16 23:34:36 ERROR Elements not present after 0.5 seconds on page: https://capital.com/nl/accounttypen
2024-08-16 23:34:36 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555e303bc6ca <unknown>
#1 0x555e3008d600 <unknown>
#2 0x555e300dcbcb <unknown>
#3 0x555e300dceb1 <unknown>
#4 0x555e30120b24 <unknown>
#5 0x555e300ff8cd <unknown>
#6 0x555e3011e04a <unknown>
#7 0x555e300ff643 <unknown>
#8 0x555e300cfd31 <unknown>
#9 0x555e300d079e <unknown>
#10 0x555e3038425b <unknown>
#11 0x555e303881f2 <unknown>
#12 0x555e30371615 <unknown>
#13 0x555e30388d82 <unknown>
#14 0x555e3035625f <unknown>
#15 0x555e303abe68 <unknown>
#16 0x555e303ac040 <unknown>
#17 0x555e303bb49c <unknown>
#18 0x7fbac7694ac3 <unknown>